package.json

```diff
@@ -112,10 +112,8 @@
     "@babel/preset-env": "7.16.7",
     "@chanzuckerberg/axe-storybook-testing": "5.0.0",
     "@storybook/addon-a11y": "6.4.18",
-    "@storybook/addon-actions": "6.4.18",
```

Comment:
No need to include. Actions is part of essentials and is installed in all new Storybooks by default.